### PR TITLE
Add support for the api object ServiceAccount

### PR DESCRIPTION
**Pull request by PerGon**
This adds support for the Kubernetes object `ServiceAccount`

### CODE REVIEW

File: k8s/models/service_account.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+
+from .common import ObjectMeta, ObjectReference, LocalObjectReference
```
Comment (mortenlj) [inline k8s/models/service_account.py:19]: Codacy found an issue: ['.common.ObjectReference' imported but unused (F401)](https://app.codacy.com/manual/fiaas/k8s/pullRequest?prid=6196640)

File: tests/k8s/test_service_account.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+    def test_updated_if_exists(self, get, put):
+        mock_response = _create_mock_response()
+        get.return_value = mock_response
+        service_account = _create_default_service_account()
+
+        from_api = ServiceAccount.get_or_create(metadata=service_account.metadata,
+                                                secrets=service_account.secrets,
+                                                imagePullSecrets=service_account.imagePullSecrets,
+                                                automountServiceAccountToken=service_account.automountServiceAccountToken)
```
Comment (mortenlj) [inline tests/k8s/test_service_account.py:52]: Codacy found an issue: [line too long (122 > 120 characters) (E501)](https://app.codacy.com/manual/fiaas/k8s/pullRequest?prid=6196640)

**Review by gregjones — APPROVED** [review]
👍

**Review by gregjones — APPROVED** [review]
👍 👍

**Review by mortenlj — APPROVED** [review]

**Comment by PerGon** [comment]
Thanks for the 👍 I had slipped a couple of lines that shouldn't have been there. They are gone now 😅 

After the final approval - Should I merge and tag? Or someone else will do it?

**Comment by mortenlj** [comment]
> After the final approval - Should I merge and tag? Or someone else will do it?

If you are allowed by github to do it, you can do it. If not, let us know and someone else will do it.

For some reason the builds are not triggering in Semaphore, and I can't see anything obviously wrong. Might be a temporary problem with Semaphore.

**Review by oyvindio — APPROVED** [review]
LGTM

**Comment by PerGon** [comment]
Superseeded by #93 - it's the same code but coming from a branch and not a fork. Thank you for your 👍

**Pull request closed without merging**